stake/asx/funding.py

```diff
@@ -79,7 +80,7 @@ def as_url_params(self) -> str:
 class FundingRecord(BaseModel):
     action: Optional[Action] = None
     amount: Optional[float] = None
-    approved_by: None
+    approvedBy: Optional[str] = None
```

Comment:
The convention here is lower_case

```suggestion
    approved_by: Optional[str] = Field(None, alias="approvedBy")
```